config/prow/Makefile

```diff
@@ -15,10 +15,8 @@
 PROJECT       := knative-tests
 PROW_GCS      := knative-prow
 PROW_HOST     := https://prow.knative.dev
-TESTGRID_GCS  := knative-testgrid
```

Comment:
Could you instead leave this here and remove the TESTGRID_GCS from prow_common.mk? It's what I meant to do but overlooked.

Removing TESTGRID_CONFIG is good though, you should continue removing it.

Reply:
I have removed both TESTGRID_GCS and TESTGRID_CONFIG from prow_common.mk, as they are only used in production Prow.

Reply:
There's a different TESTGRID_GCS in prow-staging, are you sure?

Reply:
Oh, because it has GENERATE_TESTGRID_CONFIG set to false. Okay.